app/components/header/index.js

```diff
@@ -76,6 +76,7 @@ class Header extends Component {
           className="background-image"
           src="/static/img/speakers_and_organizers.jpg"
           alt="background"
+          aria-hidden
```

Comment:
One small thing here, as per the standard it should be `aria-hidden="true"`

https://www.w3.org/TR/wai-aria-1.0/states_and_properties#aria-hidden